wrapper/src/test/java/integration/container/TestDriverProvider.java

```diff
@@ -165,6 +165,10 @@ public void beforeEach(ExtensionContext context) throws Exception {
                       try {
                         instanceIDs = auroraUtil.getAuroraInstanceIds();
                       } catch (SQLException ex) {
+                        if ("28P01".equals(ex.getSQLState())) {
```

Comment:
Could this error be thrown by RDS databases during switchover?

Reply:
It's hard to say. The goal here is to detect authentication errors (i.e. test environment errors that are rare but may happen) and stop the tests.

Reply:
We know that error 28000 (not 28P01) is raised when TAZ cluster is switching over.